L1Trigger/TrackFindingTracklet/src/VMProjectionsMemory.cc

```diff
@@ -34,9 +37,16 @@ void VMProjectionsMemory::writeVMPROJ(bool first) {
   if (first) {
     bx_ = 0;
     event_ = 1;
-    out_.open(fname.c_str());
+
+    if (not std::filesystem::exists(dirVM)) {
+      system((string("mkdir -p ") + dirVM).c_str());
```

Comment:
check if successful

Reply:
DONE
--  P.S. As this file/directory creation code appears in dozens of places, a future PR (almost ready) will move it to a single common function, to reduce duplicate code.